sql/telemetry/client_ltv/view.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE VIEW
```

Comment:
I think this should be a query rather than a view. We don't really have great capabilities for authorized views, and the other table is in a project we don't control which makes it more difficult.

Reply:
Makes sense. Just to confirm I see that as meaning:

* move this over to the `telemetry_derived`dir
*  remove ```CREATE OR REPLACE VIEW AS `moz-fx-data-shared-prod.telemetry.client_ltv`
AS```
* rename this file as `query.sql`

Let me know if you something else in mind!

Reply:
Nope, we should be fine keeping this in `revenue_derived`, we can keep that as a new dataset.

Other than that, what you said is correct. You can even try to [schedule the query yourself](https://github.com/mozilla/bigquery-etl#scheduling-queries-in-airflow)!

Reply:
`sql/revenue_derived/client_ltv_v1/query.sql` already exists (the locked down dataset). Would you prefer this go under a new directory, i.e. `sql/revenue_derived/client_ltv_reduced/query.sql`?

Reply:
Yup, that would make sense. I would perhaps call it `client_ltv_normalized`, what do you think?

Reply:
that is much better name. SGTM.